app/proxy/service.go

```diff
@@ -91,12 +91,28 @@ func (q *Query) Params() interface{} {
 
 // ParamsAsMap returns query params converted to plain map.
 func (q *Query) ParamsAsMap() map[string]interface{} {
-	if paramsMap, ok := q.Params().(map[string]interface{}); ok {
+	paramsMap := make(map[string]interface{})
+	if paramsStrings, ok := q.Params().(map[string]string); ok {
+		for k, v := range paramsStrings {
+			paramsMap[k] = v
+		}
+		return paramsMap
+	} else if paramsMap, ok := q.Params().(map[string]interface{}); ok {
```

Comment:
won't this always be true? if you are returning a `map[string]interface{}` always why would we ever try to do `map[string]string`?

Reply:
it wouldn't be true in two cases:

1. params is `nil`
2. if `map[string]string{}{"account_id": id}` was attached to params later. which I was doing. but I've changed that and now the first clause is no longer needed. thanks for prompting me to think more about this piece of code, I think it's improved now.